docs/glossary.rst

```diff
@@ -58,6 +58,12 @@ Glossary
       `~plasmapy.analysis.fit_functions.AbstractFitFunction`.  Also see
       module `~plasmapy.analysis.fit_functions`.
 
+
+   :term:`force-free`
+      In plasma physics, a **force-free magnetic field** is a magnetic field where the Lorentz force is zero, meaning the magnetic pressure greatly exceeds the plasma pressure, allowing non-magnetic forces to be neglected. This condition is often approximated in the Sun's corona.
```

Comment:
```suggestion
      In plasma physics, a **force-free magnetic field** is a magnetic
      field where the Lorentz force is zero, meaning the magnetic
      pressure greatly exceeds the plasma pressure, allowing 
      non-magnetic forces to be neglected. This condition is often
      approximated in the Sun's corona.
```
🎨 ...since we to keep the lines of text to be ≤ 72 characters

Reply:
Fair, I will set up my editor to keep with the standard :3